explorer/src/components/instruction/serum/types.ts

```diff
@@ -301,7 +301,8 @@ export function isSerumInstruction(instruction: TransactionInstruction) {
     SERUM_PROGRAM_IDS.includes(instruction.programId.toBase58()) ||
     MARKETS.some(
       (market) =>
-        market.programId && market.programId.equals(instruction.programId)
+        market.programId &&
+        market.programId.toBase58() === instruction.programId.toBase58()
```

Comment:
Any reason not to use the pubkey equals method? Using `toBase58` is pretty expensive

Reply:
Not sure why this was introduced. Thanks!